### PR TITLE
Add password authentication to FastAPI

### PR DESCRIPTION
**Pull request by knord94**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,21 +131,31 @@ storage = local
...
+The API has an option to inlcude user authentication. This can be set in the [default configuration](./src/isar/config/default.ini).
+
+
+```
+authentication_enabled = false
+authentication_enabled = true
+```
+
+By default, the `local` storage module is used and API authentication is disabled. If using Azure Blob Storage a set of environment variables must be
+available which gives access to an app registration that may use the storage account. Enabling API authentication also requires the same environment variables. The required variables are
```
Comment (autt) [inline README.md:151]:
```suggestion
By default, the `local` storage module is used and API authentication is disabled. If using Azure Blob Storage, a set of environment variables which gives access to an app registration that may use the storage account must be available. Enabling API authentication also requires the same environment variables. The required variables are
```

File: src/isar/apis/security/authentication.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        }
+
+        self.token_algorithm = token_algorithm
+        self.token_expire_minutes = token_expire_minutes
+        self.pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+
+    def verify_password(self, plain_password, hashed_password):
+        return self.pwd_context.verify(plain_password, hashed_password)
+
+    def get_password_hash(self, password):
```
Comment (autt) [inline src/isar/apis/security/authentication.py:70]: Is this function in use?

Comment (knord94) [reply src/isar/apis/security/authentication.py]: It is not, good catch!

**Review by autt — APPROVED** [review]
Looks good :tada: You should consider squashing the commits before merging.

**Pull request merged**